### PR TITLE
jgrss/dependencies

### PR DESCRIPTION
**Pull request by jgrss**
This PR pins the upper version of `pytorch-lightning` and upgrades `geowombat`.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -47,14 +49,14 @@ install_requires =
     future>=0.17.1
     tensorboard>=2.2.0
     PyYAML>=5.1
-    pytorch_lightning>=1.7.6
+    pytorch_lightning>=1.7.6,<=1.9.5
```
Comment (jgrss) [inline setup.cfg:52]: `pytorch-lightning==2.0.0` breaks some things in `cultionet`

File: setup.cfg [hunk 2]
```diff
@@ -47,14 +49,14 @@ install_requires =
...
     torchmetrics>=0.10.0,<0.11.0
     ray>=2.0.0,<=2.1.0
     gudhi>=3.7.1
     pyarrow>=11.0.0
-    geowombat@git+https://github.com/jgrss/geowombat.git@v2.1.6
+    geowombat@git+https://github.com/jgrss/geowombat.git@v2.1.8
     tsaug@git+https://github.com/jgrss/tsaug.git
-    geosample@git+https://github.com/jgrss/geosample.git
+    geosample@git+https://github.com/jgrss/geosample.git@v1.0.1
```
Comment (jgrss) [inline setup.cfg:59]: Same as previous installation -- there wasn't a release previously available.

File: setup.cfg (hunk 1 above)
Comment (nnguyen622) [reply setup.cfg:52]: I saw pytorch-lightning==2.0.0 in early March 👍

File: setup.cfg (hunk 2 above)
Comment (nnguyen622) [reply setup.cfg:59]: oh nice! Now we have geosample release

**Review by nnguyen622 — APPROVED** [review]
Thanks for updating the geowombat version 👍

**Pull request merged**